docs/internal/Element-Type-Specs.md

```diff
@@ -435,6 +435,7 @@ _Interface for uploading documents or other files_
 - **fileSizeLimit**: `number` -- maximum file size in KB (default: no limit)
 - **subfolder**: `string` -- by default, files are uploaded into a subfolder with the name of the application serial. However, this can be over-ridden by specifying this parameter. This should rarely be required.
 - **showDescription**: `boolean` -- if `true`, an additional text input will be displayed alongside each file to allow the applicant to specify a description for each file. (default `false`)
+- **showFileRestrictions**: `boolean` -- will display the allowed file extensions and maximum size below the upload box (default: `true`)
```

Comment:
I wonder if default would be better at first so we can have a more easy transition with all of the templates?

I haven't fully thought this through, but wondered if this might help.

Reply:
> I wonder if default would be better at first so we can have a more easy transition with all of the templates?

It would in the short term, but in the long run it'd be more annoying I think. 99% of the time we'd want it on (in fact I can't really think of a use case for turning it off).

And it's easy to quickly update templates in place as we see them -- Fiji is already pretty much done.